src/thorlabs_cube/driver/tcube/tsc.py

```diff
@@ -0,0 +1,346 @@
+import struct as st
+
+from thorlabs_cube.driver.base import _Cube
+from thorlabs_cube.driver.message import MGMSG, Message, MsgError
+
+
+class Tsc(_Cube):
+    """TSC001 T-Cube Motor Controller class"""
+
+    def __init__(self, serial_dev):
+        super().__init__(serial_dev)
+        self.status_report_counter = 0
+
+    async def handle_message(self, msg) -> None:
+        msg_id = msg.id
+        data = msg.data
+
+        if msg_id == MGMSG.HW_DISCONNECT:
+            raise MsgError("Error: Please disconnect the TSC001")
+        elif msg_id == MGMSG.HW_RESPONSE:
+            raise MsgError("Hardware error, please disconnect and reconnect the TSC001")
+        elif msg_id == MGMSG.HW_RICHRESPONSE:
+            (code,) = st.unpack("<H", data[2:4])
+            raise MsgError(
+                "Hardware error {}: {}".format(code, data[4:].decode(encoding="ascii"))
+            )
+        elif (
+            msg_id == MGMSG.MOT_MOVE_COMPLETED
+            or msg_id == MGMSG.MOT_MOVE_STOP
+            or msg_id == MGMSG.MOT_GET_STATUSUPDATE
+        ):
+            if self.status_report_counter == 25:
+                self.status_report_counter = 0
+                await self.send(Message(MGMSG.MOT_MOVE_COMPLETED))
+            else:
+                self.status_report_counter += 1
+            (
+                self.position,
+                self.encoder_count,
+                self.status_bits,
+                self.chan_identity_two,
+                _,
+                _,
+                _,
+            ) = st.unpack(
+                "<LLLHLLL",
+                data[2:],
+            )
+
+    async def get_bay_used(self) -> int:
+        """Identify which bay is being used by the controller on Thorlabs Hub
+
+        :return: Integer representing the bay being used on the Thorlabs Hub
+        """
+        get_msg = await self.send_request(
+            MGMSG.HUB_REQ_BAYUSED, [MGMSG.HUB_GET_BAYUSED], self._CHANNEL
```

Comment:
```suggestion
            MGMSG.HUB_REQ_BAYUSED, [MGMSG.HUB_GET_BAYUSED], Tsc._CHANNEL
```
Use class constants.